mars/tensor/arithmetic/abs.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 1999-2018 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+from ... import opcodes as OperandDef
+from ..utils import infer_dtype
+from .core import TensorUnaryOp
+from .utils import arithmetic_operand
+
+
+@arithmetic_operand(sparse_mode='unary')
+class TensorAbs(TensorUnaryOp):
+    _op_type_ = OperandDef.ABS
+    _handler_name = 'abs'
+
+    @property
+    def handler_name(self):
```

Comment:
In my opinion, `fun_name` seems a better name.

Reply:
And also, I think because `execute` etc is a classmethod of Operand, so the property of `fun_name` is unnecessary, just `cls._fun_name` seems pretty acceptable to me.